front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -155,6 +133,15 @@ export default function AddOrEditStudyModal({ editionMode, study }: Props) {
     }
   }, [isCreateStudyError, isPatchStudyError, isDeleteStudyError]);
 
+  const isDateValid = useMemo(() => {
+    const startDate = currentStudy?.start_date;
+    const earliestDate = getEarliestDate(
+      currentStudy?.expected_end_date,
+      currentStudy?.actual_end_date
+    );
+    return !!(startDate && startDate > earliestDate);
+  }, [currentStudy?.start_date, currentStudy?.expected_end_date, currentStudy?.actual_end_date]);
+
```

Comment:
Should this const be invalid also if end date is earlier than start date ?

Reply:
The back only handle the case where we enter a start date and no expected/actual end date.